resources/lang/en/validation.php

```diff
@@ -34,7 +37,7 @@
     'different'            => 'The :attribute and :other must be different.',
     'digits'               => 'The :attribute must be :digits digits.',
     'digits_between'       => 'The :attribute must be between :min and :max digits.',
-    'dimensions'           => 'The :attribute has invalid image dimensions.',
+    'dimensions'           => 'The :attribute size is invalid: must be at least '.$minImageSize['width'].'x'.$minImageSize['height'].' and no larger than '.$maxImageSize['width'].'x'.$maxImageSize['height'].'.',
```

Comment:
We can customize this with Laravel's built-in [custom error messages](https://laravel.com/docs/5.8/validation#custom-error-messages) functionality, like `digits_between` above! This will read the provided arguments from anywhere we use this `dimensions` validator: 

```suggestion
    'dimensions'           => 'The :attribute size is invalid: must be at least :min_widthx:min_height and no larger than :max_widthx:max_height.',
```

Reply:
Great find, thanks! Updated.